packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-mixed.twig

```diff
@@ -50,7 +50,11 @@
 
       <div class="ma__header__hamburger__nav-container" aria-hidden="true">
         <div class="ma__header__hamburger__logo ma__header__hamburger__logo--mobile">
-          {% include "@atoms/09-media/site-logo.twig" %}
+          {% include "@atoms/09-media/site-logo.twig" with {
+            url: '#',
+            title: 'An Absurdly Long Title for the Site Logo',

```

Comment:
Are these for testing purposes only? Also you can change the sitewide data in packages/patternlab/styleguide/source/_data/data.json instead of override the site-logo data in multiple places